templates-java/src/main/java/br/com/templates_java/config/jms_aws_sqs/ProducerMessageGenerator.java

```diff
@@ -0,0 +1,24 @@
+package br.com.templates_java.config.jms_aws_sqs;
+
+
+import br.com.generator.core.Generator;
+import br.com.generator.core.GeneratorOptions;
+
+import java.io.File;
+import java.io.IOException;
+
+public class ProducerMessageGenerator extends Generator {
+
+    private GeneratorOptions generatorOptions;
+
+    public ProducerMessageGenerator(GeneratorOptions generatorOptions) {
+        this.generatorOptions = generatorOptions;
+    }
+
+    public File runGenerate() throws IOException {
+        this.generatorOptions.setTemplatePath("/templates/config/template-producer-message.txt");
```

Comment:
Consider defining a constant for the template path `/templates/config/template-producer-message.txt` to improve maintainability and avoid hardcoded values.